CHANGELOG.md

```diff
@@ -90,6 +90,7 @@
 * [BUGFIX] Compactor: fixed "could not guess file size" log when uploading blocks deletion marks to the global location. #3807
 * [BUGFIX] Prevent panic at start if the http_prefix setting doesn't have a valid value. #3796
 * [BUGFIX] Memberlist: fixed panic caused by race condition in `armon/go-metrics` used by memberlist client. #3724
+* [CHANGE] Querier: returning 422 (instead of 500) when query hits `max_chunks_per_query` limit with block storage. #3895
```

Comment:
This should be considered a bug fix. It's not a breaking change.

```suggestion
* [BUGFIX] Querier: returning 422 (instead of 500) when query hits `max_chunks_per_query` limit with block storage. #3895
```